### PR TITLE
Dragscroll

### PR DESCRIPTION
**Pull request by MHC03**
Issue is #81. This technically works on my test construction with a number of divs and I think it should also work with the real appstore screenshots. But somehow I still got an Internal Server Error when trying to run the backend, so I would like to ask if someone else could test this on my stead.

### CODE REVIEW

**Comment by sGerli** [comment]
screenshots
It is working, but something happened to the styles, and they are getting stacked.

**Comment by MHC03** [comment]
Shoot, I forgot to float the screenshots. 
I removed some unnecessary div construction from before in above commit and the screenshots should now be floated left, so that they are not stacked. 

EDIT: Is it also possible to test this on smartphones, tablets etc. ?

File: src/components/pages/widgets/ScreenshotList.vue [hunk 1]
```diff
@@ -43,22 +58,23 @@ export default {
     max-width: 100%;
     overflow-x: scroll;
     overflow-y: hidden;
-    -webkit-overflow-scrolling: touch;
```
Comment (sGerli) [inline src/components/pages/widgets/ScreenshotList.vue]: I think we should not remove this line, it's for iOS smooth scrolling.

**Review by sGerli — CHANGES_REQUESTED** [review]
Everything else seems to be working correctly.

File: src/components/pages/widgets/ScreenshotList.vue (hunk 1 above)
Comment (sGerli) [reply src/components/pages/widgets/ScreenshotList.vue]: And if possible I also think we should remove the dragscroll-0.0.8.min.js file

**Comment by sGerli** [comment]
You also have to remove it from index.html (line 32)

**Review by sGerli — APPROVED** [review]

**Pull request merged**